apps/docs/app/features/content-menu/ContentMenu.tsx

```diff
@@ -33,23 +33,23 @@ export const ContentMenu = forwardRef((_, ref) => {
       variant="list"
       size="sm"
       allowToggle
-      mt={6}
+      marginTop={6}
```

Comment:
Endrer litt shorthand margin og padding bruk til fordel for å skrive det helt ut.